content/en/aws/managed-streaming-for-kafka/index.md

```diff
@@ -66,7 +68,155 @@ The expected output is something like the following
         },
         "NumberOfBrokerNodes": 3,
         "State": "ACTIVE",
-        "ZookeeperConnectString": "localhost:4515"
+        "ZookeeperConnectString": "localhost:4513"
     }
 }
-{{< / command >}}
\ No newline at end of file
+{{< / command >}}
+
+## Create a kafka topic
+
+In this step of using LocalStack MSK, you create a client machine. You use this client machine to create a topic that produces and consumes data.
+
+First, Install Java on the client machine by running the following command:
+
+{{< command >}}
+sudo yum install java-1.8.0
+{{< / command >}}
+
+Run the following command to download Apache Kafka.
+
+{{< command >}}
+wget https://archive.apache.org/dist/kafka/2.2.1/kafka_2.12-2.2.1.tgz
```

Comment:
I think we can combine the `wget` and `tar` commands into a single snippet, to make the description a bit more terse.

Reply:
Done